pkg/shared/oddmail/go_simple_mail_service.go

```diff
@@ -0,0 +1,85 @@
+package oddmail
+
+import (
+	"errors"
+
+	mail "github.com/xhit/go-simple-mail/v2"
+)
+
+// GoSimpleMailService is an EmailService implementation for the GoSimpleMail library
+type GoSimpleMailService struct {
+	smtpServer *mail.SMTPServer
+	smtpClient *mail.SMTPClient
```

Comment:
My main question around this package is how it works with SES. It seems like on EASi, we use the AWS SDK which seems to handle the credentialing through IAM (from what I can tell?) - seems like we'd need to get SMTP credentials for SES for this to work - do you know how this would look?

I think it's nice that we have a single client that always uses SMTP (whether thats locally for mailcatcher, or deployed with AWS SES), but I just want to make sure that this approach will work once we deploy, since it's not like how we've done it before!

Reply:
This is the documentation I was reading regarding the AWS SES SMTP interface:
https://docs.aws.amazon.com/ses/latest/dg/send-email-smtp.html

It seems straightforward to me, but it's true this is an area of risk. I suggest we do a spike dedicated to AWS SES configuration and testing the SMTP interface with our production criteria.

Reply:
I think as long as we're able to get SMTP credentials (endpoint, user, pass) we'll probably be fine. Guessing those are one-time, and we can generate them and configure/set them in SSM